modules/jinja_macros.py

```diff
@@ -6,16 +6,27 @@
 from docutils.parsers.rst import Directive, directives
 
 
+class TolerantOptionSpec(object):
+
+    # Use str to convert any and all options
+    def __getitem__(self, key):
+        return str
+
+    # Make sure docutils will use this thing
+    # even though it's empty
+    def __nonzero__(self):
+        return True
+
+
 class CallMacro(Directive):
     has_content = True
     required_arguments = 1
-    optional_arguments = 10
+    optional_arguments = 0
     final_argument_whitespace = False
-    option_spec = {}
+    option_spec = TolerantOptionSpec()
```

Comment:
It'd be easier to tell if we had tests, wouldn't it? I'll at least check that manually.